doc.go

```diff
@@ -267,4 +267,50 @@
 //     }
 //     // ...
 //   }
+//
+// Value Groups
+//
+// Dig provides value groups to allow producing and consuming many values of
+// the same type. Value groups allow constructors to send values to a named,
+// unordered collection in the container. Other constructors can request all
+// values in this collection as a slice.
+//
+// Constructors can send values into value groups by returning a dig.Out
+// struct tagged with `group:".."`.
+//
+//   type HandlerResult struct {
+//     dig.Out
+//
+//     Handler Handler `group:"server"`
```

Comment:
`Handler http.Handler`?

Reply:
No, that was intentionally not `http.Handler` because given `[]http.Handler`, you can't really register them on a `ServeMux` -- there's no pattern/name attached.